tests/capture-tests.js

```diff
@@ -1,10 +1,10 @@
-require(["mobifyjs/utils", "capture"], function(Utils, Capture) {
+require(["mobifyjs/utils", "capture"], function (Utils, Capture) {
```

Comment:
Sorry about all the whitespace changes, the test script was formatted weird, so I reformatted it.